pom.xml

```diff
@@ -228,7 +228,7 @@
         <clojure.math.numeric-tower.version>0.0.1</clojure.math.numeric-tower.version>
         <carbonite.version>1.5.0</carbonite.version>
         <snakeyaml.version>1.11</snakeyaml.version>
-        <httpclient.version>4.3.3</httpclient.version>
+        <httpclient.version>4.5</httpclient.version>
```

Comment:
I am not sure about this change. Is this dependency shaded? There has been a discussion recently on updating versions of non-shaded dependencies. 
https://www.mail-archive.com/dev@storm.apache.org/msg32949.html
If it is not shaded, it is better to upgrade the version only for storm-opentsdb module